Library/Homebrew/cmd/upgrade.rb

```diff
@@ -192,6 +192,9 @@ def upgrade_formula(f)
     end
     oh1 "Upgrading #{Formatter.identifier(f.full_specified_name)} #{upgrade_version} #{fi.options.to_a.join(" ")}"
 
+    # prefetch dependencies and bottles
+    fi.prefetch
```

Comment:
What would you think about just naming this `fetch`?

Reply:
I renamed to `fetch_and_build` for more specificity. Fun fact: did have it named `fetch` first.